zombies.js

```diff
@@ -208,6 +205,106 @@
  * @return {string/boolean}   Weapon name or false if nothing is equipped.
  */
 
+class Player {
+  constructor(name, health, strength, speed) {
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this._pack = [];
+    this._maxHealth = this.health;
+    this.isAlive = true;
+    this.equipped = false;
+  }
+
+  getPack() {
+    return this._pack;
+  }
+
+  getMaxHealth() {
+    if (!this._maxHealth) {
+      return this.health;
+    }
+    return this._maxHealth;
+  }
+
+  takeItem(item) {
+    if (this._pack.length === 3) {
```

Comment:
You should check that the `this._pack.length` is less than 3 to handle edges cases where the pack length could end up being more than 3.